x-pack/plugins/search_indices/public/application.tsx

```diff
@@ -15,22 +15,25 @@ import { QueryClientProvider } from '@tanstack/react-query';
 
 import { UsageTrackerContextProvider } from './contexts/usage_tracker_context';
 import { initQueryClient } from './services/query_client';
-import { SearchIndicesServicesContext } from './types';
+import { SearchIndicesServicesContext, SearchIndicesServicesContextDeps } from './types';
+import { Router } from '@kbn/shared-ux-router';
 
 export const renderApp = async (
-  App: React.FC<{}>,
```

Comment:
We should not make this change. If we want to render a router for `/app/elasticsearch/start` then we could pass a router to this function from the plugin.